hopglass-server.js

```diff
@@ -497,6 +498,8 @@ function getMetrics(stream) {
       counter_clients += get(n, 'statistics.clients.total')
     }
 
+    nodeNames[_.get(n, ['nodeinfo', 'node_id'])] = _.get(n, ['nodeinfo', 'hostname'])
```

Comment:
thanks for the hints! If nothing speaks against it I'd set the default to "k". "undefined" might break things in stream.write()?